gpustack/server/server.py

```diff
@@ -550,15 +550,21 @@ async def _cleanup_orphaned_gateway_data(self, session: AsyncSession):
             config=k8s_config,
         )
 
+    def _should_create_default_cluster(self) -> bool:
+        # only server or both will get into this logic
+        if self._config.server_role() == Config.ServerRole.BOTH:
+            return True
+        if self._config.token and not self._config.token.startswith(API_KEY_PREFIX):
```

Comment:
Is there a better check than the prefix check? A user provided token like `gpustack123` doesn't seem to work

Reply:
Removed the prefix check. The original design is a quick return for `_migrate_legacy_token`. If the `--token` can be set in server side, the prefix check should be removed.